app/views/support/organisations/claims/list.njk

```diff
@@ -17,7 +17,13 @@
   {% include "_includes/notification-banner.njk" %}
 
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-full">
+      {# {% include "_includes/deadline-banner.njk" %} #}
```

Comment:
If you're hiding the banner in support, it's probably a good idea to hide the whole grid row now that it is empty.